service/controller/internal/nodecontroller/resource/readylabel/create.go

```diff
@@ -0,0 +1,45 @@
+package readylabel
```

Comment:
I just realized this should be called `readyannotation`. I'll rename it tomorrow.